### ISSUE TITLE
Fetaure request: please add the option to idicate a certain programming problem cannot be implemented in a given language

### ISSUE THREAD

**Issue by Bart-B** [Deleplace/programming-idioms#57]
Some problems presented cannot be implemented natively in all langages.
However there is no way to indicate this.

Can a mechanism be implemented so that a user can indicate a certain problem cannot be implemented in a given language?
(Of course, if some user choose that option and was wrong, the indictor should be removed is another user supplies a solution after all)

Bart

**Comment by Deleplace** [comment]
Indeed, it would be useful to be able to add "negative implementations". Maybe with a block of text for explanation, node code, and a different, dark color.
